portal-kernel/src/com/liferay/portal/kernel/portlet/constants/FriendlyURLResolverConstants.java

```diff
@@ -30,6 +30,8 @@ public class FriendlyURLResolverConstants {
 
 	public static final String URL_SEPARATOR_OBJECT_ENTRY = "/l/";
 
+	public static final String URL_SEPARATOR_OBJECT_ENTRY_FOLDER = "/folder/";
```

Comment:
I have some doubts about this:

for object input we have `URL_SEPARATOR_OBJECT_ENTRY = "/l/"`
but for object input folder we have `URL_SEPARATOR_OBJECT_ENTRY_FOLDER = "/folder/";`

- I find it incongruous
- Having the word "folder" as a separator is a bit problematic for me. It could be already the name of a page/category/whatever because it is too common.

Reply:
Any suggestion?

Reply:
at least only one letter  `f` ??? (l is for "files" so ... )